lib/float.gd

```diff
@@ -44,87 +45,269 @@ end);
+##   <Oper Name="FrExp" Arg="f"/>
+##   <Oper Name="LdExp" Arg="f exp"/>
+##   <Oper Name="AbsoluteValue" Arg="f"/>
+##   <Oper Name="Norm" Arg="f"/>
+##   <Oper Name="Hypothenuse" Arg="x y"/>
+##   <Oper Name="Frac" Arg="f"/>
+##   <Oper Name="SinCos" Arg="f"/>
+##   <Oper Name="Erf" Arg="f"/>
+##   <Oper Name="Zeta" Arg="f"/>
+##   <Oper Name="Gamma" Arg="f"/>
+##   <Description>
+##     These are all standard math functions, directly copied from the standard math library; see <C>man 3 math</C> for their list.
+##   </Description>
+## </ManSection>
+## <#/GAPDoc>
+##
+DeclareAttribute("Cos", IsFloat);
+DeclareAttribute("Sin", IsFloat);
+DeclareAttribute("Tan", IsFloat);
+DeclareAttribute("Sec", IsFloat);
+DeclareAttribute("Csc", IsFloat);
+DeclareAttribute("Cot", IsFloat);
+DeclareAttribute("Asin", IsFloat);
+DeclareAttribute("Acos", IsFloat);
+DeclareAttribute("Atan", IsFloat);
+DeclareAttribute("Cosh", IsFloat);
+DeclareAttribute("Sinh", IsFloat);
+DeclareAttribute("Tanh", IsFloat);
+DeclareAttribute("Sech", IsFloat);
+DeclareAttribute("Csch", IsFloat);
+DeclareAttribute("Coth", IsFloat);
+DeclareAttribute("Asinh", IsFloat);
+DeclareAttribute("Acosh", IsFloat);
+DeclareAttribute("Atanh", IsFloat);
+DeclareOperation("Log", [IsFloat]);
+DeclareAttribute("Log2", IsFloat);
+DeclareAttribute("Log10", IsFloat);
+DeclareAttribute("Log1p", IsFloat);
+DeclareAttribute("Exp", IsFloat);
+DeclareAttribute("Exp2", IsFloat);
+DeclareAttribute("Exp10", IsFloat);
+DeclareAttribute("Expm1", IsFloat);
+DeclareAttribute("CubeRoot", IsFloat);
+DeclareAttribute("Square", IsFloat);
+DeclareAttribute("Ceil", IsFloat);
+DeclareAttribute("Floor", IsFloat);
+DeclareAttribute("Round", IsFloat);
+DeclareAttribute("Trunc", IsFloat);
+DeclareOperation("Atan2", [IsFloat, IsFloat]);
 DeclareAttribute("FrExp", IsFloat);
-DeclareOperation("LdExp", [IsFloat,IsInt]);
-DeclareAttribute("Argument", IsFloat);
+DeclareOperation("LdExp", [IsFloat, IsInt]);
 DeclareAttribute("AbsoluteValue", IsFloat);
 #DeclareAttribute("Norm", IsFloat); #already defined
-DeclareOperation("Hypothenuse", [IsFloat,IsFloat]);
-DeclareAttribute("Frac",IsFloat);
-DeclareAttribute("SinCos",IsFloat);
-DeclareAttribute("Erf",IsFloat);
-DeclareAttribute("Zeta",IsFloat);
-DeclareAttribute("Gamma",IsFloat);
-DeclareAttribute("ComplexI",IsFloat);
-
-DeclareAttribute("PrecisionFloat",IsFloat);
-DeclareAttribute("SignFloat",IsFloat);
+DeclareOperation("Hypothenuse", [IsFloat, IsFloat]);
+DeclareAttribute("Frac", IsFloat);
+DeclareAttribute("SinCos", IsFloat);
+DeclareAttribute("Erf", IsFloat);
+DeclareAttribute("Zeta", IsFloat);
+DeclareAttribute("Gamma", IsFloat);
+#DeclareAttribute("ComplexI", IsFloat); #obsolete
 
-DeclareAttribute("Sup", IsFloat);
-DeclareAttribute("Inf", IsFloat);
-DeclareAttribute("Mid", IsFloat);
-DeclareAttribute("AbsoluteDiameter", IsFloat);
-DeclareAttribute("RelativeDiameter", IsFloat);
-#DeclareOperation("Diameter", IsFloat);
-DeclareOperation("Overlaps", [IsFloat,IsFloat]);
```

Comment:
So `Overlaps` is removed, too, but this isn't noted in the commit message either.